### PR TITLE
Mysql configuration for play-java-ebean-example

### PR DESCRIPTION
**Pull request by kevinobama**
Mysql configuration for play-java-ebean-example

I did sign the CLA.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,13 @@ EBean is a Java ORM library that uses SQL:
 and the documentation can be found here:
 
 [https://ebean-orm.github.io/](https://ebean-orm.github.io/)
+
+## Mysql configuration
```
Comment (cchantep) [inline README.md:27]: I don't see where that's specific to this example? If not would rather have a link yo the already existing doc about how to configure a DB

**Review by cchantep — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kevinobama) [reply README.md:27]: The link is https://www.playframework.com/documentation/2.6.x/JavaDatabase,  if  we can write it here. It's more convenient for new user.

Comment (kevinobama) [reply README.md:27]: Should we add this link to this document?

Comment (cchantep) [reply README.md:27]: As for other doc link there, if the one to the Play doc is not sufficient.

Comment (kevinobama) [reply README.md:27]: ok

**Comment by kevinobama** [comment]
ok.

**Pull request closed without merging**